code/modules/power/singularity/containment_field.dm

```diff
@@ -72,7 +72,9 @@
 /obj/machinery/field/containment/proc/on_entered(datum/source, atom/movable/AM)
 	SIGNAL_HANDLER
 	if(isliving(AM))
-		shock(AM)
+		var/mob/living/L = AM
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
okay...

Reply:
Working around the 5 - 10 y/o code surly didn't help me to realize you had updated the contrib docs. It's a good guideline tho all things considered. Thanks mothblocks